.github/workflows/black-fix.yml

```
@@ -0,0 +1,37 @@
+name: black
+
+on: [push, pull_request]
```

```suggestion
on:
  push:
    paths:
      - '**.py'
```

Running on `pull_request` as well means it can trigger twice and we get a race condition as both jobs try to commit.